easybuild/easyconfigs/f/FACE/FACE-1.1.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'FACE'
+version = '1.1.1'
+
+homepage = 'https://github.com/szaghi/FACE'
+description = """A Fortran Ansi Colors (and Styles) Environment.
+A KISS pure Fortran Library for easy colorize (and stylize) strings."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
```
$ file FACE/1.1.1-GCCcore-10.3.0/include/face.mod
FACE/1.1.1-GCCcore-10.3.0/include/face.mod: gzip compressed data, from Unix, original size 3041
```

So, this should move to `GCC`.

Reply:
Fixed, had forgotten to push this since I made that change locally a long time ago...